backend/src/main/java/ai/verta/modeldb/versioning/RepositoryDAORdbImpl.java

```diff
@@ -437,7 +467,22 @@ private void deleteRoleBindingsOfAccessibleResources(List<RepositoryEntity> allo
               workspaceDTO.getWorkspaceType(),
               false,
               null);
-      list.forEach((o) -> builder.addRepositories(((RepositoryEntity) o).toProto()));
+      Map<String, RepositoryEntity> result = new LinkedHashMap<>();
+      for (Object o : list) {
```

Comment:
NIT:  Object repoObj and if you do not mind change list to repoList

Reply:
fixed